Desktop app builds and launches but renders no UI

## Summary

The packaged macOS desktop app builds and launches but renders **no UI at all**. The process stays alive, the signature verifies, and the window exists — but nothing is on screen and the agent never becomes usable.

## Reproduce

```bash
bun run --cwd packages/app-core/platforms/electrobun build
ditto <build>/Eliza-dev.app /Applications/Eliza-dev.app
open -a /Applications/Eliza-dev.app
```

Reproduced on `develop` (`883fb83be9` and again 429 commits later at `f40f930a7e`), macOS 26.5.1, arm64, Bun 1.3.14.

## Observed

Five distinct defects, all the same shape — **a package ships without something it imports eagerly**:

1. **Shell dies before creating a window.** `fused-wake.ts` statically imports `@elizaos/plugin-local-inference/voice-wake`, which `electrobun.config.ts:582` marks `external`. The packaged `app/bun/index.js` has no `node_modules` on its resolution path:
   ```
   error: Cannot find module '@elizaos/plugin-local-inference/voice-wake'
     from '.../Resources/app/bun/index.js'
   ```
   The app code runs inside a `Worker` in `main.js`, and the launcher never observes its failure — so the parent survives on an empty AppKit event loop and *looks* healthy (alive process, 0% CPU, valid signature) while producing no window, tray icon, or agent.

2. **Agent crash-loops.** `isRuntimePluginPackage()` matches **any** scoped `plugin-*` package, so `@octokit/plugin-request-log`, `@jimp/plugin-*`, and `@milkdown/plugin-*` are excluded from the runtime bundle while their dependents ship:
   ```
   [eliza] Failed to start CLI: Cannot find module '@octokit/plugin-request-log'
     from '.../eliza-dist/node_modules/@octokit/rest/dist-src/index.js'
   ```
   Five restart attempts, then it gives up.

3. **Always-loaded plugins are not bundled.** `MOBILE_VIEW_PLUGINS` (`viewEveryPlatform: true`) are loaded on every platform but appear in no bundling list, so `plugin-personal-assistant`'s lifeops routes throw on the missing `@elizaos/plugin-calendar`.

4. **Hard dependencies dropped.** The transitive-walk filter drops every non-always-bundled plugin — including hard `dependencies` of packages that are themselves shipping (`plugin-blocker`, `-goals`, `-phone`, `-reminders`, `-x`).

   **3 and 4 are the direct cause of the blank UI.** The failed post-ready app-route tail pins `/api/health` `startup.phase` at `"degraded"`; the desktop boot gate (`isRuntimeReadyFromBootProgress`) advances only on `phase === "running"`, so the renderer's startup coordinator sits in `starting-runtime` forever and never mounts.

5. **Overlay renders off-screen.** With the runtime healthy the chat overlay is a viewport-centred drawer (`sm:top-1/2 -translate-y-1/2`, `h-[min(640px,80vh)]`), but the resting bar window is ~140px tall — so it lays out at **`y = -50`** and is clipped away entirely. Measured live:
   ```
   {"winH":140,"overlay":{"y":-50,"h":120},"ovlVis":"visible"}
   ```
   The DOM is present and interactive the whole time, which is exactly why this reads as "the app opened but nothing is there".

## Expected

Per `packages/app-core/platforms/electrobun/docs/desktop-window-lifecycle.md`: a chromeless bottom bar with the home pill at rest, expanding into the chat overlay on click.

## Acceptance criteria

- [ ] Packaged app boots with `/api/health` reporting `startup.phase: "ready"` and `app-route-tail: "complete"`
- [ ] Zero `Cannot find module` errors in the packaged runtime
- [ ] Home pill renders at rest and the chat overlay opens unclipped (`y >= 0`, `bottom <= winH`)
- [ ] Regression tests cover each defect class
- [ ] `bun run verify` passes

## Environment

macOS 26.5.1 (25F80), arm64 · Bun 1.3.14 · Node 24.19.0 · Electrobun 1.18.1

--- a/packages/app-core/platforms/electrobun/src/surface-windows.test.ts
+++ b/packages/app-core/platforms/electrobun/src/surface-windows.test.ts
@@ -139,6 +139,29 @@ describe("SurfaceWindowManager app windows", () => {
     expect(fixture.manager.listWindows("settings")).toEqual([first]);
   });
 
+  it("navigates an already-open settings window to the requested tab instead of just focusing it (#19996)", async () => {
+    const fixture = createFixture();
+
+    await fixture.manager.openSettingsWindow("open-settings-voice");
+    const window = fixture.created[0];
+    expect(window).toBeDefined();
+    expect(window?.webview.loadURL).not.toHaveBeenCalled();
+
+    // Menu action while the window is already open: the section must change.
+    await fixture.manager.openSettingsWindow("open-settings-permissions");
+
+    expect(fixture.created).toHaveLength(1);
+    expect(window?.webview.loadURL).toHaveBeenCalledWith(
+      "http://127.0.0.1:5173/?shell=settings&tab=permissions",
+    );
+    expect(window?.focus).toHaveBeenCalled();
+
+    // No tab hint (plain "Settings Window" menu item): focus only, no reload.
+    window?.webview.loadURL.mockClear();
+    await fixture.manager.openSettingsWindow();
+    expect(window?.webview.loadURL).not.toHaveBeenCalled();
+  });
+
   it("opens browser surfaces with browse query encoding and ignores browse for non-browser surfaces", async () => {
     const fixture = createFixture();
 

--- a/packages/app-core/platforms/electrobun/src/surface-windows.ts
+++ b/packages/app-core/platforms/electrobun/src/surface-windows.ts
@@ -232,6 +232,17 @@ export class SurfaceWindowManager {
       (entry) => entry.surface === "settings",
     );
     if (existing) {
+      // A tab hint on an already-open Settings window must navigate the
+      // window to the requested section — the menu items ("Voice Controls",
+      // "Permissions", "Cloud Settings", …) all route through here, and
+      // merely re-focusing left the stale section on screen (#19996).
+      const normalizedTab = normalizeSettingsTabHint(tabHint);
+      if (normalizedTab && existing.window.webview.loadURL) {
+        const rendererUrl = await this.resolveRendererUrlFn();
+        existing.window.webview.loadURL(
+          buildSurfaceWindowRendererUrl(rendererUrl, "settings", normalizedTab),
+        );
+      }
       existing.window.focus();
       return this.toSnapshot(existing);
     }

